data-access/mandataris-download.ts

```diff
@@ -0,0 +1,257 @@
+  bestuursorgaanInTijdId: string | null,
+  sort: { ascOrDesc: 'ASC' | 'DESC'; filterProperty: string } | null,
+): Promise<Array<{ [key: string]: string }>> {
+  let sortFilter: string | null = null;
+  let bestuursorgaanInTijdFilter: string | null = null;
+
+  if (sort) {
+    sortFilter = `
+      ORDER BY ${sort.ascOrDesc}(${sort.filterProperty})
+    `;
+  }
+
+  if (bestuursorgaanInTijdId) {
+    bestuursorgaanInTijdFilter = `?bestuursorgaanInTijd mu:uuid ${sparqlEscapeString(
+      bestuursorgaanInTijdId,
+    )}.`;
+  }
+
+  const escapedUriValues = mandatarisUris
+    .map((uri) => sparqlEscapeUri(uri))
+    .join('\n');
+  const queryString = `
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>
+    PREFIX org: <http://www.w3.org/ns/org#>
+    PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+    PREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>
+    PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+    PREFIX regorg: <https://www.w3.org/ns/regorg#>
+
+    SELECT DISTINCT ?mandataris ?mandaatLabel ?publicatieStatusLabel ?rangorde (GROUP_CONCAT(DISTINCT ?beleidsdomeinLabel; SEPARATOR="; ") AS ?beleidsdomeinen) ?statusLabel ?fName ?lName ?start ?einde ?fractieLabel ?bestuursorgaanLabel
+    WHERE {
+      VALUES ?mandataris { ${escapedUriValues} }
+      ?mandataris a mandaat:Mandataris.
+      ?mandataris org:holds ?mandaat.
+      ?mandataris mandaat:status ?status.
+      ?mandataris mandaat:start ?start.
+
+      ?status skos:prefLabel ?statusLabel.
+
+      ?mandaat org:role ?bestuursfunctie.
+      ?bestuursfunctie skos:prefLabel ?mandaatLabel.
+
+      ?mandaat ^org:hasPost ?bestuursorgaanInTijd.
+      ?bestuursorgaanInTijd mandaat:isTijdspecialisatieVan ?bestuursorgaan. 
+      ${bestuursorgaanInTijdFilter ?? ''}
+      ?bestuursorgaan skos:prefLabel ?bestuursorgaanLabel.
+
+      ?mandataris mandaat:isBestuurlijkeAliasVan ?persoon.
+
+      OPTIONAL {
+        ?mandataris lmb:hasPublicationStatus ?publicatieStatusCode.
+        ?publicatieStatusCode skos:prefLabel ?publicatieStatusLabel.
+      }
+      
+      OPTIONAL {
+        ?mandataris mandaat:rangorde ?rangorde.
+      }
+    
+      OPTIONAL {
+        ?mandataris mandaat:beleidsdomein ?beleidsdomeinCode.
+        ?beleidsdomeinCode skos:prefLabel ?beleidsdomeinLabel.
+      }
+
+      OPTIONAL {
+        ?persoon persoon:gebruikteVoornaam ?fName.
+      }
+      OPTIONAL {
+        ?persoon foaf:familyName ?lName.
+      }
+
+      OPTIONAL {
+        ?mandataris org:hasMembership ?lidmaatschap.
+        ?lidmaatschap org:organisation ?fractie.
+        ?fractie regorg:legalName ?fractieLabel.
+      }
+      OPTIONAL {
+        ?mandataris mandaat:einde ?einde.
+      }
+    }
+    ${sortFilter ?? ''}
+  `;
+
+  const sparqlResult = await query(queryString);
+
+  return getSparqlResults(sparqlResult).map((result) => {
+    return {
```

Comment:
can you move this logic to the jsonToCsv function and reuse it in the verkiezingsresultaten route? You can generalize this to a map over the keys of the result and taking the value as '' if undefined